lib/libbol.py

```diff
@@ -1362,6 +1383,20 @@ def to_bytes(self) -> bytes:
         self.write(f)
         return f.getvalue()
 
+    def add_respawn(self, rsp:JugemPoint):
```

Comment:
Nit: Space between colon and type hint. (I thought you'd spot this one too, so I didn't point it out.)